api/envoy/service/cluster/v2alpha/aggregated_event_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package envoy.service.cluster.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.service.cluster.v2alpha";
+option java_outer_classname = "AggregatedEventServiceProto";
+option java_multiple_files = true;
+option java_generic_services = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.service.cluster.v3";
+
+// [#protodoc-title: gRPC Aggregated Event Service]
+
+// [#not-implemented-hide:]
+// Service for streaming different types of events from Envoy to a server. The examples of
+// such events may be health check or outlier detection events.
+service AggregatedEventService {
```

Comment:
My $0.02 is for the naming `EventReportingService` (analogos to Load Reporting Service (LRS)). The reason AggregatedEventService might not be great is that it might be used in non-aggregated (today) and aggregated (future) ways on the same stream. If that sounds awful, `ProxyEventService`? IDK.

Reply:
`EventReportingService` sgtm +1

Reply:
done